`_restclient` tests failing.

The `_restclient` tests are failing. Looks like a pretty simple fix. I'll submit a PR.

```console
==================================== ERRORS ====================================
_______________________ ERROR at setup of test_build_url _______________________
file /home/runner/work/hydrotools/hydrotools/python/_restclient/tests/test_restclient.py, line 146
  def test_build_url(loop):
E       fixture 'loop' not found
>       available fixtures: _session_event_loop, aiohttp_client, aiohttp_client_cls, aiohttp_raw_server, aiohttp_server, basic_test_server, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, event_loop, event_loop_policy, monkeypatch, naked_server, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, temp_sqlite_db, tests/test_restclient.py::<event_loop>, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/home/runner/work/hydrotools/hydrotools/python/_restclient/tests/test_restclient.py:146
______ ERROR at setup of test_restclient_nest_asyncio_ModuleNotFoundError ______
file /home/runner/work/hydrotools/hydrotools/python/_restclient/tests/test_restclient.py, line 159
  def test_restclient_nest_asyncio_ModuleNotFoundError(loop):
E       fixture 'loop' not found
>       available fixtures: _session_event_loop, aiohttp_client, aiohttp_client_cls, aiohttp_raw_server, aiohttp_server, basic_test_server, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, event_loop, event_loop_policy, monkeypatch, naked_server, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, temp_sqlite_db, tests/test_restclient.py::<event_loop>, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/home/runner/work/hydrotools/hydrotools/python/_restclient/tests/test_restclient.py:159
=========================== short test summary info ============================
ERROR python/_restclient/tests/test_restclient.py::test_build_url
ERROR python/_restclient/tests/test_restclient.py::test_restclient_nest_asyncio_ModuleNotFoundError
========================= 79 passed, 2 errors in 1.87s =========================
```

--- a/python/_restclient/tests/test_restclient.py
+++ b/python/_restclient/tests/test_restclient.py
@@ -143,10 +143,10 @@ def test_get_headers_have_precedent_over_instance(basic_test_server):
         assert all(k_v_pair in r.headers.items() for k_v_pair in method_headers.items())
 
 
-def test_build_url(loop):
+def test_build_url(event_loop):
     base_url = "http://www.test.gov/"
     query_params = {"key": "value"}
-    with RestClient(enable_cache=False, loop=loop) as client:
+    with RestClient(enable_cache=False, loop=event_loop) as client:
 
         assert client.build_url(base_url) == base_url
         assert client.build_url(base_url, query_params) == f"{base_url}?key=value"
@@ -156,7 +156,7 @@ class ModuleFoundError(Exception):
     ...
 
 
-def test_restclient_nest_asyncio_ModuleNotFoundError(loop):
+def test_restclient_nest_asyncio_ModuleNotFoundError(event_loop):
     """Test for #99. Ensure ModuleNotFoundError raised if `nest_asyncio` not installed"""
     import asyncio
     import warnings
@@ -183,4 +183,4 @@ def test_restclient_nest_asyncio_ModuleNotFoundError(loop):
                 # this test will need to change if `nest_asyncio` becomes a requirement
                 RestClient(enable_cache=False)
 
-    loop.run_until_complete(test())
+    event_loop.run_until_complete(test())

